src/Shift-ClassInstaller-Tests/ShClassInstallerTest.class.st

```diff
@@ -141,6 +141,16 @@ ShClassInstallerTest >> testDuplicateClassPreserveSlots [
 	
 ]
 
+{ #category : #tests }
+ShClassInstallerTest >> testDuplicatedInstanceVariable [
+
+	superClass := self newClass:#ShCITestClass1 slots:#(aSlot).
+	newClass := self newClass:#ShCITestClass2 superclass: superClass slots:#().
+	
+	self should: [newClass := self newClass:#ShCITestClass2 superclass: superClass slots:#(aSlot)] raise: Error.
```

Comment:
more concrete exception class than Error should be tested here

Reply:
I will improve the test.

Thanks